dtests/src/test/scala/io/snappydata/hydra/cdcConnector/DmlOpsInBatchCdcConnectorTest.scala

```diff
@@ -0,0 +1,714 @@
+        if (snappyHome == null) {
+            throw new Exception("SNAPPY_HOME should be set as an environment variable")
+        }
+        homeDir = System.getProperty("user.home")
+        testDir = s"$snappyHome/../../../dtests/src/resources/scripts/cdcConnector"
+        appJar = s"$snappyHome/../../../snappy-poc/cdc/target/cdc-test-0.0.1.jar"
+        println("Snappy home : " + snappyHome)
+        println("Home Dir : " + homeDir)
+        println("Test Config Dir : " + testDir)
+        println("App Jar location : " + appJar)
+
+        setDMLMaxChunkSize(50L)
+        before()
+
+    }
+
+    override def afterAll(): Unit = {
+         after()
+
+    }
+
+    def setDMLMaxChunkSize(size: Long): Unit = {
+        GemFireXDUtils.DML_MAX_CHUNK_SIZE = size
+    }
+
+    protected def getUserAppJarLocation(jarName: String, jarPath: String): String = {
+        var userAppJarPath: String = null
+        if (new File(jarName).exists) jarName
+        else {
+            val baseDir: File = new File(jarPath)
+            try {
+                val filter: IOFileFilter = new WildcardFileFilter(jarName)
+                val files: util.List[File] = FileUtils.listFiles(baseDir, filter,
+                    TrueFileFilter.INSTANCE).asInstanceOf[util.List[File]]
+                println("Jar file found: " + util.Arrays.asList(files))
+                import scala.collection.JavaConverters._
+                for (file1: File <- files.asScala) {
+                    if (!file1.getAbsolutePath.contains("/work/") ||
+                        !file1.getAbsolutePath.contains("/scala-2.10/")){
+                        userAppJarPath = file1.getAbsolutePath
+                    }
+
+                }
+            } catch {
+                case e: Exception =>
+                    println("Unable to find " + jarName
+                        + " jar at " + jarPath + " location.")
+            }
+            userAppJarPath
+        }
+    }
+    
+    def truncateSqlTable(): Unit = {
+
+        // scalastyle:off classforname Class.forName
+
+        Class.forName("com.microsoft.sqlserver.jdbc.SQLServerDriver")
+        val conn = DriverManager.getConnection(
+            "jdbc:sqlserver://sqlent.westus.cloudapp.azure.com:1433;DatabaseName=testdatabase",
+            "sqldb", "snappydata#msft1")
+        conn.createStatement().execute("delete from [testdatabase].[dbo].[ADJUSTMENT] " +
+            "where (adj_id between 95000010061 and 950000100110)")
+        Thread.sleep(50000)
+        conn.createStatement().execute("truncate table testdatabase.cdc.dbo_ADJUSTMENT_CT")
+
+        val sqlDF = conn.createStatement(
+            ResultSet.TYPE_SCROLL_INSENSITIVE, ResultSet.CONCUR_READ_ONLY)
+            .executeQuery("select * from [testdatabase].[dbo].[ADJUSTMENT] " +
+                "where (adj_id between 95000010061 and 950000100110)")
+
+        if (sqlDF.next) System.out.println("FAILURE : The result set should have been empty")
```

Comment:
Where are these sysouts going? Should log it some file so that debugging can be easy.